packages/fcl/src/discovery/services.js

```diff
@@ -0,0 +1,33 @@
+import {config} from "@onflow/sdk"
+import {invariant} from "@onflow/util-invariant"
+
+const asyncPipe = (...fns) => input => fns.reduce((chain, fn) => chain.then(fn), Promise.resolve(input))
+
+async function addServices(services = []) {
+  const endpoint = await config.first(["discovery.authn.api"])
```

Comment:
Could use `config.get("discovery.authn.api")` if there is only one key to get from

Reply:
should there be a default perhaps? suppose fallback here could be testnet `"https://fcl-discovery.onflow.org/api/testnet/authn"`

Reply:
Yeah I had the same thought, but I'm not sure I do. I might rather it have to be explicit so they don't develop, push to prod, then realize they are still using testnet because they never set up their config. But I'm curious to hear other thoughts.

Reply:
yeah, explicit may be best, with the invariant check